apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -131,6 +131,27 @@ window.PLOrderBlocks = function (uuid, options) {
       placePairingIndicators();
       setAnswer();
     },
+    start: function () {
+      if (enableIndentation) {
+        $(dropzoneElementId)[0].style.background =
+          'linear-gradient(#000, #000) no-repeat border-box, '.repeat(+maxIndent + 1).slice(0, -2);
```

Comment:
> 'linear-gradient(#000, #000) no-repeat border-box

This seems really complicated to just draw a line, is there no simpler way? I'm not super familiar with this part of CSS but my guess is that there is a simpler way. I could be wrong but it would be nice to know at least.

Reply:
This was taken straight from the original issue #4921. I could check if there are simpler ways of drawing a line. If there is anyone else who knows a more elegant way, please do chime in 👀

Reply:
you should be able to just set the background to be a specific color instead of using the `linear-gradient` function. That seems very unneccessary. I'm also not sure what `no-repeat` and `border-box` do or if they are necessary.

I would also prefer it to have some opacity so it looks more gray instead of just the strong black color.